tests/components/ziggo_next/test_config_flow.py

```diff
@@ -0,0 +1,14 @@
+"""Test the Ziggo Next config flow."""
+from homeassistant import data_entry_flow
+from homeassistant.config_entries import SOURCE_USER
+from homeassistant.core import HomeAssistant
+
+
+async def test_show_user_form(hass: HomeAssistant) -> None:
```

Comment:
Please fill out this test so its testing the setup of the entry.

see `tests/components/harmony/test_config_flow.py` for an example